verilog/formatting/token_annotator_test.cc

```diff
@@ -2692,6 +2693,121 @@ TEST(TokenAnnotatorTest, AnnotateFormattingWithContextTest) {
   }
 }
 
+struct AnnotateBreakAroundCommentsTestCase {
+  FormatStyle style;
+
+  int left_token_enum;
+  std::string left_token_string;
+
+  std::string whitespace_between;
+
+  int right_token_enum;
+  std::string right_token_string;
+
+  InitializedSyntaxTreeContext context;
+  ExpectedInterTokenInfo expected_annotation;
+};
+
+TEST(TokenAnnotatorTest, AnnotateBreakAroundComments) {
+  const std::initializer_list<AnnotateBreakAroundCommentsTestCase> kTestCases = {
+      {
+          // No comments
+          DefaultStyle,
+          '=', "=",
+          "   ",
+          yytokentype::TK_DecNumber, "0",
+          {/* unspecified context */},
+          {1, SpacingOptions::Undecided}
+      },
+      {
+          // 0 // comment
+          DefaultStyle,
+          yytokentype::TK_DecNumber, "0",
+          "   ",
+          yytokentype::TK_EOL_COMMENT, "// comment",
+          {/* unspecified context */},
+          {2, SpacingOptions::MustAppend}
+      },
+      {
+          // 0// comment
+          DefaultStyle,
+          yytokentype::TK_DecNumber, "0",
+          "",
+          yytokentype::TK_EOL_COMMENT, "// comment",
+          {/* unspecified context */},
+          {2, SpacingOptions::MustAppend}
+      },
+      {
+          // 0 \n  // comment
+          DefaultStyle,
+          yytokentype::TK_DecNumber, "0",
```

Comment:
Suggest adding a test cases for `;` and `,` and `KEYWORD` on the left, which are very common.

Reply:
Added test cases for several keywords.